docker-compose.yml

```
@@ -90,30 +58,11 @@ services:
       - ./stac_fastapi:/app/stac_fastapi
       - ./scripts:/app/scripts
     command: >
-      bash -c "./scripts/wait-for-it.sh app-sqlalchemy:8081 && cd stac_fastapi/sqlalchemy && alembic upgrade head && python /app/scripts/ingest_joplin.py http://app-sqlalchemy:8081"
-    depends_on:
-      - database
-      - app-sqlalchemy
-
-  # Load joplin demo dataset into the PGStac Application
-  loadjoplin-pgstac:
-    image: stac-utils/stac-fastapi
-    environment:
-      - ENVIRONMENT=development
-    volumes:
-      - ./stac_fastapi:/app/stac_fastapi
-      - ./scripts:/app/scripts
-    command:
-      - "./scripts/wait-for-it.sh"
-      - "app-pgstac:8082"
-      - "--"
-      - "python"
-      - "/app/scripts/ingest_joplin.py"
-      - "http://app-pgstac:8082"
+      bash -c "./scripts/wait-for-it.sh stac-fastapi-sqlalchemy:8081 && alembic upgrade head && python /app/scripts/ingest_joplin.py http://stac-fastapi-sqlalchemy:8081"
```

```suggestion
      bash -c "./scripts/wait-for-it.sh -t 30 stac-fastapi-sqlalchemy:8081 && alembic upgrade head && python /app/scripts/ingest_joplin.py http://stac-fastapi-sqlalchemy:8081"
```

This is not necessarily related to this PR, but this almost always times out for me when starting from a fresh slate. The service seems to become available after about 19 or 20 seconds, so increasing this timeout to 30s should cover us.